test/pg-cdc-resumption/configure-materalize.td

```diff
@@ -8,17 +8,27 @@
 # by the Apache License, Version 2.0.
 
 > DROP SOURCE IF EXISTS mz_source CASCADE;
+> DROP SECRET IF EXISTS pgpass CASCADE;
+> DROP CONNECTION IF EXISTS pg CASCADE;
```

Comment:
Oh, derp--this shouldn't be necessary! Looks like we missed adding this diff:

```diff
diff --git a/src/testdrive/src/action/sql.rs b/src/testdrive/src/action/sql.rs
index b9b58f807..1e129c2c7 100644
--- a/src/testdrive/src/action/sql.rs
+++ b/src/testdrive/src/action/sql.rs
@@ -27,9 +27,9 @@ use mz_ore::retry::Retry;
 use mz_ore::str::StrExt;
 use mz_pgrepr::{Interval, Jsonb, Numeric};
 use mz_sql_parser::ast::{
-    CreateClusterReplicaStatement, CreateClusterStatement, CreateDatabaseStatement,
-    CreateSchemaStatement, CreateSecretStatement, CreateSourceStatement, CreateTableStatement,
-    CreateViewStatement, Raw, ReplicaDefinition, Statement, ViewDefinition,
+    CreateClusterReplicaStatement, CreateClusterStatement, CreateConnectionStatement,
+    CreateDatabaseStatement, CreateSchemaStatement, CreateSecretStatement, CreateSourceStatement,
+    CreateTableStatement, CreateViewStatement, Raw, ReplicaDefinition, Statement, ViewDefinition,
 };
 
 use crate::action::{Action, ControlFlow, State};
@@ -140,6 +140,13 @@ impl Action for SqlAction {
                 )
                 .await
             }
+            Statement::CreateConnection(CreateConnectionStatement { name, .. }) => {
+                self.try_drop(
+                    &mut state.pgclient,
+                    &format!("DROP CONNECTION IF EXISTS {} CASCADE", name),
+                )
+                .await
+            }
             _ => Ok(()),
         }
```

Line above should already be unnecessary.

Reply:
Ah, you're totally right that this was missed, but also this testdrive script runs with `--no-reset` so the logic there doesn't apply. So I applied your patch and left the code in question in place!

Reply:
ah til